data/get_syn_data.py

```diff
@@ -212,7 +218,7 @@ def generate_json(include_at_risk_populations, include_released_only, do_not_dow
             column_order = schema_columns
 
             # Uuid is now called Id, but ofc there are still some old files with
-            # Uuid column name 😭
+            # Uuid column name 
```

Comment:
you don't like the sobbing emoji :)?

Reply:
Ah sorry this file is not supposed to be uploaded in the PR, just some local changes when I do the debugging. 
And I like the sobbing emoji haha